org.eclipse.swt.win32.win32.x86/src/org/eclipse/swt/widgets/Text.d

```diff
@@ -33,6 +33,13 @@ import org.eclipse.swt.widgets.Control;
 
 import java.lang.all;
 
+version(Tango){
+    import tango.text.convert.Utf;
```

Comment:
No need to add new code which suppors Tango.

Reply:
I removed it.